io.vikunja.Desktop.appdata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.vikunja.Desktop</id>
+  <name>Vikunja</name>
+  <summary>The open-source, self-hostable to-do app. Organize everything, on all platforms.</summary>
+  <project_license>AGPLv3</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">io.vikunja.Desktop</launchable>
```

Comment:
this is an invalid launchable. it should be the name of a .desktop file.

Reply:
Ah, wasn't sure about that. Will fix.

Reply:
Fixed.

Reply:
Well this could be my bad though I'm sure I included `.desktop` in the original